hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/PeriodicService.java

```diff
@@ -167,21 +167,18 @@ protected synchronized void startPeriodic() {
     stopPeriodic();
 
     // Create the runnable service
-    Runnable updateRunnable = new Runnable() {
-      @Override
-      public void run() {
-        LOG.debug("Running {} update task", serviceName);
-        try {
-          if (!isRunning) {
-            return;
-          }
-          periodicInvoke();
-          runCount++;
-          lastRun = Time.now();
-        } catch (Exception ex) {
-          errorCount++;
-          LOG.warn(serviceName + " service threw an exception", ex);
+    Runnable updateRunnable = () -> {
+      LOG.debug("Running {} update task", serviceName);
+      try {
+        if (!isRunning) {
+          return;
         }
+        periodicInvoke();
+        runCount++;
+        lastRun = Time.now();
+      } catch (Exception ex) {
+        errorCount++;
+        LOG.warn(serviceName + " service threw an exception", ex);
```

Comment:
Use {}

Reply:
Thanks for your help reviewing the code, I will fix it.